### PR TITLE
cephfs-top: fix Refresh interval

### PR DESCRIPTION
**Pull request by joscollin**
make refresh interval consistent

### CODE REVIEW

File: doc/cephfs/cephfs-top.rst [hunk 1]
```diff
@@ -72,11 +72,11 @@ By default, `cephfs-top` connects to cluster name `ceph`. To use a non-default c
 
   $ cephfs-top --cluster <cluster>
 
-`cephfs-top` refreshes stats every second by default. To chose a different refresh interval use::
+`cephfs-top` refreshes stats every second by default. To choose a different refresh interval use::
 
   $ cephfs-top -d <seconds>
 
-Interval should be greater or equal to 0.5 second. Fractional seconds are honoured.
```
Comment (vshankar) [inline doc/cephfs/cephfs-top.rst:79]: I do not understand what's wrong with having refresh interval >= 0.5 seconds?

Comment (joscollin) [reply doc/cephfs/cephfs-top.rst:79]: It should be >=0.5 seconds [here](https://github.com/ceph/ceph/blob/master/src/tools/cephfs/top/cephfs-top#L548) then. It's confusing whether 0.5 is inclusive or not. We could fix the error text instead.

Comment (vshankar) [reply doc/cephfs/cephfs-top.rst:79]: Yeah, fix the help text instead.

File: src/tools/cephfs/top/cephfs-top [hunk 2]
```diff
@@ -415,8 +415,8 @@ class FSTop(object):
 if __name__ == '__main__':
     def float_greater_than(x):
         value = float(x)
-        if value < MIN_REFRESH_INTERVAL:
-            raise argparse.ArgumentTypeError(f'{value} should be greater than '
+        if value <= MIN_REFRESH_INTERVAL:
+            raise argparse.ArgumentTypeError(f'Refresh interval should be greater than '
```
Comment (vshankar) [inline src/tools/cephfs/top/cephfs-top:419]: ... and keep this error text fix too.

**Comment by joscollin** [comment]
@vshankar did the changes, please check

**Comment by vshankar** [comment]
jenkins test make check

**Comment by joscollin** [comment]
jenkins test make check

**Review by vshankar — APPROVED** [review]

**Pull request merged**